bin/cyhy-tool

```diff
@@ -262,9 +262,9 @@ def main():
 
         for orgId in orgIds:
             retire(db, orgId)
-        print "Do not forget to move " + str(
+        print str(
             orgIds
-        ) + ' to the "Retired CyHy Orgs" tab in the Google Spreadsheet!'
+        ) + " has been successfully retired."
```

Comment:
I overlooked the fact that this is outputting the list of retired orgs, so potentially plural (even though in reality, it's probably almost always just one org being retired at a time).  Anyway, this language somewhat works for either case:
```suggestion
        ) + " have been successfully retired."
```